data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			branch delayedretire
+				not "free worlds reconciliation"
+			`	He leans in close to you and says quietly, "I know I said I was retiring, Terry convinced me to clean up my mess first. I am atoning for my failures, do look after her, she's a very loyal colleague."`
+			label delayedretire
+			`	After which he finally, actually, departs for good.`
+				decline
+
+#This gets picked up silently on Hai-home when landing for response 2 - leads to go kill pirates mission with chat on Allhome.
+mission "Hai Leaks Response 3"
+	name "Meet with Security Forces on Allhome"
+	description "Meet with the Security Forces on Allhome before fighting the pirates."
+	landing
+	source "Hai-home"
+	destination "Allhome"
+	to offer
+		has "Hai Leaks Response 2: done"
+
+#Go kill pirates mission with chat on Allhome.
+mission "Hai Leaks Response 4"
+	name "Defeat the Black Lich"
+	description "Defeat Black Lich's Marauder Leviathan at Waypoint."
+	autosave
+	landing
+	source "Allhome"
+	waypoint "Waypoint"
+	destination "Allhome"
+	to offer
+		has "Hai Leaks Response 3: done"
+	on offer
+		conversation
+			`When you land on Allhome you are met by the Hai Captain of the local forces. You're not an expert at judging Hai ages but he appears to be a little younger than expected.`
+			`	"Captain <last>! I heard you were coming to assist."`
+			choice
+				`	"Time to route some pirates baby!"`
+				`	"I'm here to reinforce you, what can we do?"`
+			`	"Uh well, yes. I presume you saw the pirates on your way in? They've been appearing en masse and overhwelming security forces to raid technology and goods from the Hai traders and their security. Some of them have even been sneaking down onto the planet and capturing Hai, taking them away to goodness only knows where."`
+			choice
+				`	"I had heard about the kidnapping."`
+			`	He nods, "Unfortunately, it's taking a while for them to redistribute an appropriately resourced defence force thanks to the Unfettered."`
+			`	"Even more unfortunately," he continues, "it seems that the overabundance of these pirates is a result of one particular individual's success. His ship, called the Black Lich, is hanging out in Waypoint with the stolen Geocoris that we hear might have been converted into some kind of semi-stable weapons platform and now be being used as a temporary base."`
+			`	"Fortunately this gives us an opportunity though. If the kidnapped Hai are still on board we might be able to rescue them, and if we can defeat the Black Lich we may stem the tide of opportunists that are swarming through the wormhole."`
+			`	"What do you say?"`
+			`	The poor Captain seems desperate.`
+			choice
+				`	"We shall bring them safely home."`
```

Comment:
You might consider mentioning more explicitly, both in the mission description and this conversation, that the ship must be captured to free the prisoners.